assets/src/scripts/outputs-viewer/components/Toast/Toast.jsx

```diff
@@ -1,8 +1,6 @@
 import React from "react";
 import { ToastContainer } from "react-toastify";
 
-import "react-toastify/dist/ReactToastify.min.css";
```

Comment:
["No need to import the css file anymore"](https://github.com/fkhadra/react-toastify/releases/tag/v11.0.0)